### DISCUSSION TITLE
Parsing xml File and then storing the values in nested json structure

### DISCUSSION THREAD

**Discussion by Usamamalik11** [nlohmann/json#2645] [Q&A] [answered]
Hi there, I wanted to ask if there is any way that we can store the parsed values from xml in the form of nested json structure. I have seen almost all documentation and issues but I am unable to find a solution. I'll be grateful if someone can give me any suggestion.( I am pretty noob in c++). Thank You.

**Comment by nlohmann** [comment, marked as answer]
No, this library only supports JSON. See also https://github.com/nlohmann/json/issues/847
